config/initializers/apipie.rb

```diff
@@ -37,5 +37,6 @@
     A lot of these schemas aren't overly specific to the actions they are listed under,
     e.g. they always say that an `id` is required but that isn't the case when
     new data is being posted to a `create` call.
-    eos
+  eos
+  config.validate                = false
```

Comment:
It looks like this has defaulted to implicit for a while, does something in the gem upgrade need it?

Reply:
I believe the new version of the gem started validating stuff by default and we just never set everything properly for the validations to work.